pay-api/src/pay_api/services/payment_account.py

```diff
@@ -308,6 +308,8 @@ def _save_account(cls, account_request: Dict[str, any], payment_account: Payment
         payment_account.auth_account_name = account_request.get('accountName', None)
         payment_account.bcol_account = account_request.get('bcolAccountNumber', None)
         payment_account.bcol_user_id = account_request.get('bcolUserId', None)
+        payment_account.pad_tos_accepted_by = account_request.get('pad_tos_accepted_by', None)
```

Comment:
why is this in snake_case when the schema is in camelCase ?

Reply:
changed